core/README.md

```diff
@@ -61,13 +61,31 @@ Baseline components are required for all other automation areas.
 
 ## Container components
 
-1. Install the container build layer to your target. In this case, you
-   install the container build for the reference distribution named **Poky**.
+1. Install the container build layer to your target.  The script
+   naming convention is
+   `setup_ci_container_&lt;distro&rt;[.&lt;release&rt;].sh`.
```

Comment:
Have you viewed this rendered? The PR renderer is leaving the literal `&lt;` and `&rt;` in.

Reply:
ok it's due to the backtick literal. Will clean.